From: Jesse Barnes <jbarnes@virtuousgeek.org>
Date: Mon, 22 Sep 2008 20:11:55 +0000 (-0700)
Subject: Add Cappuccino SlimPRO SP625F to no LVDS quirks list
X-Git-Url: http://gitweb.freedesktop.org/?p=xorg/driver/xf86-video-intel.git;a=commitdiff;h=10909d9b665864bda2b1654de009d556cd068726

Add Cappuccino SlimPRO SP625F to no LVDS quirks list

Looks like this platform might contain VBTs that indicate an LFP is present
even though it isn't.  Intended to fix bz #11368.
---

--- a/src/i830_quirks.c
+++ b/src/i830_quirks.c
@@ -229,6 +229,9 @@ static i830_quirk i830_quirk_list[] = {
     { PCI_CHIP_I965_GM, 0xa0a0, SUBSYS_ANY, quirk_ignore_lvds },
     { PCI_CHIP_I965_GM, 0x8086, 0x1999, quirk_ignore_lvds },
 
+    /* Cappuccino SlimPRO SP625F, bz #11368 */
+    { PCI_CHIP_I855_GM, 0x8086, 0x3582, quirk_ignore_lvds },
+
     /* Apple Mac mini has no lvds, but macbook pro does */
     { PCI_CHIP_I945_GM, 0x8086, 0x7270, quirk_mac_mini },
 
